src/components/Components/CandidateSection/CandidateSection.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import "./CandidateSection.scss"
+import ViewMoreIcon from "./../../assets/viewMoreIcon.svg"
```

Comment:
Just a small correction, that the ViewMoreIcon import name should use camelCase too, e.g. `viewMoreIcon`